features/product/managing_product_variants/generating_product_variant_without_value.feature

```diff
@@ -0,0 +1,17 @@
+@managing_product_variants
+    Feature: Generating product variants without any values
+        In order to sell variants without any values of a single product
```

Comment:
I think it should be something like `In order to not be able to generate variants without any options values available`